R/utils-json.R

```diff
@@ -122,21 +144,22 @@ getJSON <- function(.pd, evilMode) {
     completeCasesTable <- veupathUtils::setAttrFromList(completeCasesTable, attr)
   }
 
+  namedAttrList$variable <- new("VariableMetadataList")
   if ('xAxisVariable' %in% names(namedAttrList)) {
-    namedAttrList$xVariableDetails <- makeVariableDetails(NULL, namedAttrList$xAxisVariable$variableId, namedAttrList$xAxisVariable$entityId,  namedAttrList$xAxisVariable$displayLabel)
+    namedAttrList$variable[[1]] <- makeVariableMetadataObject(namedAttrList$xAxisVariable)
```

Comment:
(for line 148)

Will the `.pd` object have an attribute called `xAxisVariable` any more?  Seems like it might not?

Reply:
it wont, utils are updated now as well